Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -414,10 +414,13 @@ extension TestListViewController: TestRunDelegate {
         displayTestRunAlert = true
     }
     
-    //TODO this method will be updated to take a status and unless the status is "in progress" the run will be removed.
-    func removeUpdatedItemFromTable() {
-        updatedRunName = testRunList.testRunList[selectedRunIndex].name
-        testRunList.testRunList.remove(at: selectedRunIndex)
-        self.testList.reloadData()
+    // Remove the run from the list unless the status is 'in progress'
+    func removeUpdatedItemFromTable(status:String) {
+        if status != "INPROGRESS"{
```

Comment:
A run with the status "not run" will still have the status "not run" after we submit the run if we don't update any of the step statuses. So we need to make sure that we are not removing "NOT_RUN" status as well as "INPROGRESS"